src/egon/data/datasets/chp/__init__.py

```diff
@@ -310,6 +314,125 @@ def insert_biomass_chp(scenario):
     session.commit()
 
 
+def insert_chp_statusquo():
+    cfg = config.datasets()["chp_location"]
+
+    # import data for MaStR
+    mastr = pd.read_csv(
+        WORKING_DIR_MASTR_NEW / "bnetza_mastr_combustion_cleaned.csv"
+    )
+
+    mastr = mastr.loc[mastr.ThermischeNutzleistung > 0]
+
+    mastr = mastr.loc[
+        mastr.Energietraeger.isin(
+            [
+                "Erdgas",
+                "Mineralölprodukte",
+                "andere Gase",
+                "nicht biogener Abfall",
+                "Braunkohle",
+                "Steinkohle",
+            ]
+        )
+    ]
+
+    mastr.Inbetriebnahmedatum = pd.to_datetime(mastr.Inbetriebnahmedatum)
+    mastr.DatumEndgueltigeStilllegung = pd.to_datetime(
+        mastr.DatumEndgueltigeStilllegung
+    )
+    mastr = mastr.loc[mastr.Inbetriebnahmedatum <= "2019-12-31"]

```

Comment:
maybe it makes sense to refer to this [value](https://github.com/openego/powerd-data/blob/dev/src/egon/data/datasets.yml#L309) in the datasets.yaml just to keep track and know where we filter by this - might be useful for the update functionality...

Reply:
Done in [c4dfc62](https://github.com/openego/powerd-data/pull/24/commits/c4dfc62e088ec41fdf97ed2b0cb3a8c4cd7301d4)